release/nightly_tests/dataset/ray_sgd_training_compute_no_gpu.yaml

```diff
@@ -20,8 +20,8 @@ head_node_type:
 worker_node_types:
     - name: memory_node
       instance_type: i3.8xlarge
-      min_workers: 20
```

Comment:
@rkooo567 Sang, do you remember what may be the reason that we request 20 worker nodes when the test only needs 16?

Reply:
Not sure. I don't think I added this test...